conftest.py

```diff
@@ -92,6 +93,13 @@ def pytest_collection_modifyitems(config, items):
             item.add_marker(rc_only_mark)
 
 
+@pytest.hookimpl(hookwrapper=True)
+def pytest_runtest_logreport(report):
+    result = yield
+    testutils.github.update_issue(report)
+    return result
```

Comment:
I don't understand why `return yield`?

Reply:
`yield` in a keyword to yield to the caller. This can produce results in fact. So what is here is

```py
return result
```

not `return yield`.

This is done in case the wrapping `pytest_runtest_logreport()` returns something. See also https://docs.pytest.org/en/stable/writing_plugins.html#hookwrapper-executing-around-other-hooks

Reply:
Then again: It is not expected to return anything... so no idea... will remove.

Reply:
Thanks for the pointers to the documentation, it is clearer to me know.